README.md

```diff
@@ -1 +1,53 @@
-# To-Do-List
\ No newline at end of file
+# To-Do-List
+
+> This project is a presentation of working with webpack.
+
+# Web Preview
+
+![screenshot](...)
+
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## Live Demo
+
+[Live Demo](https://chaw-bot.github.io/To-Do-List/)
+
+# Install Instruction
+1. Open your terminal and `cd` to a proper location.
+2. Clone the Repo on you computer using `git clone git@github.com:chaw-bot/To-Do-List.git`
+3. Make sure you have JavaScript installed; use `js -v` .
+4. Cd to the project directory`cd To-Do-List` .
+5. Open `index.html`.
+

```

Comment:
[OPTIONAL]
-  I would highly recommend to update your README as for now screenshot is not visible, live link is broken, and also there is some missing steps like `npm install` and `npm start` from README file. Think about when recruiter visit your profile and will see the broken image and link, it will not make a good impression on that. Please fix that. :slightly_smiling_face: